chrome_manifest_v3/views/options.js

```
@@ -34,17 +34,14 @@ function save_options() {
-    function () {
-      // Update status to let user know options were saved.
-      var status = document.getElementById('status');
-      status.innerHTML = 'Options Saved.';
-      setTimeout(function () {
-        status.innerHTML = '&nbsp;';
-      }, 2500);
-    }
-  );
+  chrome.storage.sync.set({ debugMode, dueShortcuts }, function () {
+    // Update status to let user know options were saved.
+    var status = document.getElementById('status');
```

Biome (1.9.4)

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole body of the function. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'const' instead.


(lint/style/noVar)